changes.d/5738.feat.md

```diff
@@ -0,0 +1 @@
+Added sequentially checked xtriggers option to restrict checking, and the spawning behavior of any associated parentless tasks. `wall_clock` is now sequential by default.
```

Comment:
I find that a bit hard to follow. Is this better?:

```suggestion
Optionally spawn parentless xtriggered tasks sequentially - i.e., one at a time, after the previous xtrigger is satisfied, instead of all at once out to the runahead limit.  The `wall_clock` xtrigger is now sequential by default.
```